packages/puppeteer-core/src/node/ChromeLauncher.ts

```diff
@@ -266,3 +272,26 @@ function convertPuppeteerChannelToBrowsersChannel(
       return BrowsersChromeReleaseChannel.CANARY;
   }
 }
+
+/**
+ * Extracts all features from the given command-line flag
+ * (e.g. `--enable-features`, `--enable-features=`).
+ *
+ * Example input:
+ * ["--enable-features=NetworkService,NetworkServiceInProcess", "--enable-features=Foo"]
+ *
+ * Example output:
+ * ["NetworkService", "NetworkServiceInProcess", "Foo"]
+ *
+ * @internal
+ */
+export function getFeatures(flag: string, options: string[] = []): string[] {
+  const opts = options
```

Comment:
Why do we need the check at the end I think just doing this does the same, no?
```suggestion
  return options
```